### PR TITLE
Quickstart UX study article

### PR DESCRIPTION
**Pull request by DavidCBerry13**
New article design for developer quickstart ux study being run in conjunction with Azure SDK team.

### CODE REVIEW

**Comment by DavidCBerry13** [comment]
#sign-off

File: docs/azure/storage/storage-quickstart-blobs-dotnet.md [hunk 1]
```diff
@@ -0,0 +1,423 @@
+---
+title: Accessing Azure Blob Storage from .NET applications
```
Comment (IEvangelist) [inline docs/azure/storage/storage-quickstart-blobs-dotnet.md:2]:
```suggestion
title: Access Azure Blob Storage from .NET applications
```

File: docs/azure/storage/storage-quickstart-blobs-dotnet.md [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+description: Learn how to connect your applications to Azure Blob Storage
+ms.date: 06/26/2021
+ms.topic: article
+ms.custom: devx-track-dotnet
+ms.author: daberry
+ROBOTS: NOINDEX
+---
+
+# Accessing Azure Blob Storage from .NET applications
```
Comment (IEvangelist) [inline docs/azure/storage/storage-quickstart-blobs-dotnet.md:11]:
```suggestion
# Access Azure Blob Storage from .NET applications
```

File: docs/azure/storage/storage-quickstart-blobs-dotnet.md [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+```csharp
+public IEnumerable<BlobInfoModel> ListBlobsInContainer(string containerName)
+{
+    BlobContainerClient containerClient = _blobServiceClient.GetBlobContainerClient(containerName);
+
+    if (!containerClient.Exists())
+        throw new ApplicationException($"Cannot list blobs in container '{containerName}' as it does not exists");
+
+    Pageable<BlobItem> blobs = containerClient.GetBlobs();
+    var models = blobs.Select(b => new BlobInfoModel()
```
Comment (IEvangelist) [inline docs/azure/storage/storage-quickstart-blobs-dotnet.md:294]:
```suggestion
    var models = blobs.Select(b => new BlobInfoModel
```

File: docs/azure/storage/storage-quickstart-blobs-dotnet.md [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+
+```csharp
+    public void UploadBlob(string containerName, string blobName, string contentType, Stream content)
+    {
+        BlobContainerClient containerClient = _blobServiceClient.GetBlobContainerClient(containerName);
+        if (!containerClient.Exists())
+            throw new ApplicationException($"Unable to upload blobs to container '{containerName}' as the container does not exists");
+
+        BlobClient blobClient = containerClient.GetBlobClient(blobName);
+        var options = new BlobUploadOptions() { HttpHeaders = new BlobHttpHeaders() { ContentType = contentType } };
```
Comment (IEvangelist) [inline docs/azure/storage/storage-quickstart-blobs-dotnet.md:324]:
```suggestion
        var options = new BlobUploadOptions { HttpHeaders = new BlobHttpHeaders { ContentType = contentType } };
```

File: docs/azure/storage/storage-quickstart-blobs-dotnet.md [hunk 5]
```diff
@@ -0,0 +1,423 @@
...
+    {
+        BlobContainerClient containerClient = _blobServiceClient.GetBlobContainerClient(containerName);
+        if (!containerClient.Exists())
+            throw new ApplicationException($"Unable to get blob {blobName} in container '{containerName}' as the container does not exists");
+
+        BlobClient blobClient = containerClient.GetBlobClient(blobName);
+        if (!blobClient.Exists())
+            throw new ApplicationException($"Unable to delete blob {blobName} in container '{containerName}' as no blob with this name exists in this container");
+                   
+        return new BlobModel()
```
Comment (IEvangelist) [inline docs/azure/storage/storage-quickstart-blobs-dotnet.md:348]:
```suggestion
        return new BlobModel
```

**Review by IEvangelist — COMMENTED** [review]
Hi @DavidCBerry13 - I'm curious, is there a specific reason we're not using the async APIs? This is network I/O and I'm very hopeful that most real-world consumers would be using the async versions. As such, it's probably best to show them, do you agree?

**Comment by DavidCBerry13** [comment]
> Hi @DavidCBerry13 - I'm curious, is there a specific reason we're not using the async APIs? This is network I/O and I'm very hopeful that most real-world consumers would be using the async versions. As such, it's probably best to show them, do you agree?

@IEvangelist - It was really just to keep this as simple as possible for the pilot.  I think for the real article I'll probably convert everything to async

**Review by IEvangelist — APPROVED** [review]

**Comment by gewarren** [comment]
@DavidCBerry13 Can you please add this article to the table of contents so it's not orphaned?

**Pull request merged**